server/server.js

```diff
@@ -63,6 +63,38 @@ app.get('/friends', (req, res) => {
   })
 });
 
+app.get('/userdata', (req, res) => {
```

Comment:
This looks like it might be able to replace the 'user' one I had, will try to replace mine with this one once it's merged.